src/handlers/store-transactions/handler.test.ts

```diff
@@ -14,7 +14,7 @@ beforeEach(() => {
   process.env = { ...OLD_ENV };
   mockPutS3.mockClear();
   process.env.STORAGE_BUCKET = "store";
-  process.env.TRANSACTIONS_FOLDER = "btm_transactions";
+  process.env.LEGACY_TRANSACTIONS_FOLDER = "btm_transactions";
```

Comment:
When will we be able to ditch this?

Reply:
That's the next ticket that's in progress (BTM-487), where all this stuff is removed.